index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Portfolio-Mobile-First</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="wrapper">
+            <header>
+                <div class="logo">Welcome</div>
+                <button class="button-header">
+                    <img class="icone-clos" src="Images/Vector icon.png" alt="-vector">
+                </button>
+            </header>
+            <main>
+                <section class="headline">
+                  <h1 class="title">
+                    Hey there. I'm <br >
+                    Jacinta <br>
+                    I'm a software <br/>
+                    developer
+                  </h1>
+                  <p class="details">
+                    I can help you build a product , feature
+                    or website Look through some of my work and experience! If you like
+                    what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                  </p>
+                  
+                  <ul class="social-media">
+                    <li>
+                      <a href="#"><img src="Images/twitter.png" alt="Twitter" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/Linkedin icon.png" alt="linkedin" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/medium.png" alt="medium" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/github.png" alt="github" /></a>
+                    </li>
+                    <li>
+                      <a href="#"><img src="Images/angellist.png" alt="angellist" /></a>
+                    </li>
+                  </ul>
+                </section>
+        
+        </div>
```

Comment:
[Required] Your social icons are overflowing to the right. And your background image is not like the one in the template.

| Your Design | Template|
|-------------|---------|
|![port1](https://user-images.githubusercontent.com/62938735/198270427-0a1d7cdd-ee08-497e-8a6f-84fe9e241af9.gif)|![image](https://user-images.githubusercontent.com/62938735/198271289-fd689311-56fb-44a0-a3e1-fa688e51176a.png)|